datasource.go

```diff
@@ -24,14 +24,17 @@ package sdk
 type Datasource struct {
 	ID                uint        `json:"id"`
 	OrgID             uint        `json:"orgId"`
+	Uid               string      `json:"uid"`
 	Name              string      `json:"name"`
 	Type              string      `json:"type"`
+	TypeLogoUrl       string      `json:"typeLogoUrl"`
```

Comment:
```suggestion
	TypeLogoURL       string      `json:"typeLogoUrl"`
```
same here (https://github.com/grafana-tools/sdk/blob/e7b85bb25aa9694012fa6f829a2eed879e5a33ba/rest-request.go#L40)